### ISSUE TITLE
StandardDataFormats.StorageItems example does not reference StorageItems?

### ISSUE THREAD

**Issue by citelao** [MicrosoftDocs/winrt-api#1793]
I'm reading about [StandardDataFormats](https://docs.microsoft.com/en-us/uwp/api/windows.applicationmodel.datatransfer.standarddataformats?view=winrt-19041), and I'm trying to understand StorageItems.

Unfortunately, the description of "StorageItem format" is unclear to me, and the example doesn't seem to mention StorageItems.

Is there a better example? [This MSDN social thread](https://social.msdn.microsoft.com/Forums/en-US/1cb633a4-c963-48ad-b02f-6dd5e5d21915/what-is-a-storageitem) indicates similar confusion.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 05920318-494d-66dc-876a-b8e42e4c8b5a
* Version Independent ID: 39a72edd-7edd-7943-f4b1-228deea03c5b
* Content: [StandardDataFormats.StorageItems Property (Windows.ApplicationModel.DataTransfer) - Windows UWP applications](https://docs.microsoft.com/en-us/uwp/api/windows.applicationmodel.datatransfer.standarddataformats.storageitems?view=winrt-19041#Windows_ApplicationModel_DataTransfer_StandardDataFormats_StorageItems)
* Content Source: [winrt-api-build/xml/Windows.ApplicationModel.DataTransfer/StandardDataFormats.xml](https://github.com/MicrosoftDocs/winrt-api/blob/docs/winrt-api-build/xml/Windows.ApplicationModel.DataTransfer/StandardDataFormats.xml)
* Product: **uwp**
* Technology: **winrt-reference**
* GitHub Login: @mcleanbyron
* Microsoft Alias: **mcleans**

**Comment by mcleanbyron** [comment]
I'm sorry for the delay in responding to this issue. Basically, the StorageItems property returns a static string identifier that indicates that the type of data object in a data transfer operation (for example, drag and drop, share, or Clipboard) is a file or folder. 

I updated the StandardDataFormats and StorageItems articles with new code examples that better illustrate this, and I also revised the description for StorageItems to be more clear. Finally, I also added links to the [Clipboard sample](https://github.com/Microsoft/Windows-universal-samples/tree/master/Samples/Clipboard) and the [ShareTarget sample](https://github.com/microsoft/Windows-universal-samples/tree/master/Samples/ShareTarget), which also illustrate a variety of StandardDataFormats values in use.
